pyproject.toml

```diff
@@ -65,27 +65,46 @@ source = ["src"]
 omit = ["*/.tox/*", "*/__main__.py", "*/setup.py", "*/.venv*/*", "*/venv*/*"]
 
 [tool.coverage.report]
-fail_under = 77
+#fail_under = 80
 show_missing = true
 skip_covered = false
 
 [tool.tox]
 legacy_tox_ini = '''
 [tox]
-envlist = py38, py39
+envlist = py{38, 39}-{integration, unit},
```

Comment:
Could you document this in the readme as well, if it stays like this when PR is ready.

Reply:
Wrote some.  But I feel that the readme should have a proper rework.